source/elements/oneMKL/source/domains/blas/iamax.rst

```diff
@@ -165,6 +167,10 @@ iamax (USM Version)
    incx
       The stride of vector ``x``.
 
+   base
```

Comment:
Looks like this is missing in Syntax for USM Version.

Reply:
Fixed, thanks!